data/common/distances/CountryDistances.py

```diff
@@ -0,0 +1,122 @@
+class CountryDistances:
+    def __init__(self, distances):
+        self.distances = self._convert_distances(distances)
+        self._all_countries = set()
+        self._update_all_countries()
+
+    self_distance = {
+        "road": 500,
+        "sea": None,
+        "air": 500,
+    }
```

Comment:
Not entirely sure about this, should self_distance (the distance from countryA to countryA) be that ?

Reply:
I'm slightly surprised that air and road distances are the same but tbh the final impact ratio for these is marginal so we probably shouldn't care too much

Reply:
(otherwise 500km for default transportation within a same country is what's currently used in the Elm app)